### PR TITLE
feat(ACM how to): promote project import in Studio

### PR DESCRIPTION
**Pull request by NathalieC**
* promote link to the project on the studio's Welcome page

### CODE REVIEW

File: md/use-bonita-acm.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-* in a Bundle, using the setup tool, reach for the `bonita-tenant-sp-custom.properties` file (in `platform_conf/current/tenants/<TENAND_ID>/tenant_engine` folder in server has already been started or `platform_conf/initial/tenant_template_engine` before the first start of the server) and add this line `com.bonitasoft.engine.plugin.acm.REGISTER_ACM_HANDLER=true`.  
+* in a Bundle, using the setup tool, reach for the `bonita-tenant-sp-custom.properties` file (in `platform_conf/current/tenants/<TENAND_ID>/tenant_engine` folder if the server has already been started or `platform_conf/initial/tenant_template_engine` before the first start of the server) and add this line `com.bonitasoft.engine.plugin.acm.REGISTER_ACM_HANDLER=true`.  
+
+## ACM project example to import in Bonita Studio
+
+On Bonita Studio Welcome page, in the "Resources" tile, click on "Import Adaptive Case Management example".   
+This opens the "Import .bos" wizard and retrieves the project from the Internet.  
+Choose the Bonita project in which the .bos will be imported in the studio, and import.
+Then open the Credit Card Dispute application descriptor, click on the application URL, and read the Tutorial.
```
Comment (rbioteau) [inline md/use-bonita-acm.md]: Maybe just use the Deploy... displayed at the end of the import

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: md/use-bonita-acm.md (hunk 1 above)
Comment (NathalieC) [reply md/use-bonita-acm.md]: Sure! Editing that. Thanks.

File: md/use-bonita-acm.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ It can be achieved:
 On Bonita Studio Welcome page, in the "Resources" tile, click on "Import Adaptive Case Management example".   
 This opens the "Import .bos" wizard and retrieves the project from the Internet.  
 Choose the Bonita project in which the .bos will be imported in the studio, and import.
-Then open the Credit Card Dispute application descriptor, click on the application URL, and read the Tutorial.
+At the end of the importation, open the Credit Card Dispute application, and just start reading the Tutorial.
```
Comment (rbioteau) [inline md/use-bonita-acm.md]: You need to Deploy after the import :)

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: md/use-bonita-acm.md [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
-* in a Bundle, using the setup tool, reach for the `bonita-tenant-sp-custom.properties` file (in `platform_conf/current/tenants/<TENAND_ID>/tenant_engine` folder in server has already been started or `platform_conf/initial/tenant_template_engine` before the first start of the server) and add this line `com.bonitasoft.engine.plugin.acm.REGISTER_ACM_HANDLER=true`.  
+* in a Bundle, using the setup tool, reach for the `bonita-tenant-sp-custom.properties` file (in `platform_conf/current/tenants/<TENAND_ID>/tenant_engine` folder if the server has already been started or `platform_conf/initial/tenant_template_engine` before the first start of the server) and add this line `com.bonitasoft.engine.plugin.acm.REGISTER_ACM_HANDLER=true`.  
+
+## ACM project example to import in Bonita Studio
+
+On Bonita Studio Welcome page, in the "Resources" tile, click on "Import Adaptive Case Management example".   
+This opens the "Import .bos" wizard and retrieves the project from the Internet.  
+Choose the Bonita project in which the .bos will be imported in the studio, and import.
+At the end of the importation, open the Credit Card Dispute application, and just start reading the Tutorial.
```
Comment (rbioteau) [inline md/use-bonita-acm.md]:
```suggestion
At the end of the importation, click on _Deploy..._, select all artifacts and _Deploy_. At the end of the deployment open the Credit Card Dispute application, and just start reading the Tutorial.
```

**Pull request merged**